orca-core/src/test/groovy/com/netflix/spinnaker/orca/batch/pipeline/util/PackageInfoSpec.groovy

```diff
@@ -26,7 +26,7 @@ import spock.lang.Unroll
 class PackageInfoSpec extends Specification {
 
   @Subject
-  PackageInfo info = new PackageInfo(Mock(Stage), 'deb', '_', false, true, Mock(ObjectMapper))
+  PackageInfo info = new PackageInfo(Mock(Stage), 'deb', '_', true, true, Mock(ObjectMapper))
```

Comment:
No, but this forces the `parseBuildInfoUrl` method to be called (the one I've edited). Previously that method was not tested through here.